src/components/policies/search-form.tsx

```diff
@@ -0,0 +1,41 @@
+import { FormEventHandler } from 'react';
```

Comment:
This not a new component. It was extracted from the previous `src/pages/policies/policies.tsx` file